DataFormats/PatCandidates/src/PackedCandidate.cc

```diff
@@ -159,22 +161,12 @@ float pat::PackedCandidate::dz(const Point &p) const {
 
 void pat::PackedCandidate::unpackTrk() const {
     maybeUnpackBoth();
-    reco::TrackBase::CovarianceMatrix m;
-//    m(0,0)=0.5e-4/pt()/pt(); //TODO: tune
-//    m(1,1)=6e-6; //TODO: tune 
-//    m(2,2)=1.5e-5/pt()/pt(); //TODO: tune
-    m(0,0)=dptdpt_/pt()/pt(); //TODO: tune
-    m(1,1)=detadeta_; //TODO: tune 
-    m(2,2)=dphidphi_/pt()/pt(); //TODO: tune
-    m(2,3)=dphidxy_;
-    m(3,2)=dphidxy_;
-    m(4,1)=dlambdadz_;
-    m(1,4)=dlambdadz_;
-    m(3,3)=dxydxy_;
-    m(3,4)=dxydz_;
-    m(4,3)=dxydz_;
-    m(4,4)=dzdz_;
     math::RhoEtaPhiVector p3(ptTrk(),etaAtVtx(),phiAtVtx());
+    for(int i=0;i<5;i++)
```

Comment:
Resetting the elements to 0 is a race condtion. Multiple threads can be in `unpackTrk` simultaneously. One could just be leaving `unpackCovariance()` while another is in the clearing loop. The thread clearing could then pause before getting to `unpackCovariance()` and then another thread could ask for an element and would get 0.